core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -10259,6 +10260,7 @@ protected PaginatedSearchResult doGetUserList(Condition condition, String profil
         return new PaginatedSearchResult();
     }
 
+    //TODO - This method is required for a few other methods to use (found under getUserList and etc)
     protected UniqueIDPaginatedSearchResult doGetUserListWithID(Condition condition, String profileName, int limit,
```

Comment:
This method is required for a few other methods to use (found under getUserList and etc). Therefore it cannot be merged with the below method